### PR TITLE
Errormsg

### PR DESCRIPTION
**Pull request by niraliabhani**
Error handling task : https://3.basecamp.com/3537281/buckets/4129302/todos/1975372803

### CODE REVIEW

File: buildvu-client.js [hunk 1]
```diff
@@ -87,22 +87,31 @@
...
-                                    failure("Connection error");
+                        xhr.onreadystatechange = function (e) {
+                                if (xhr.readyState === 4) {
+                                    if (xhr.status === 200) {
+                                        if (!params.parameters.callbackUrl || progress || success) {
+                                            doPoll(JSON.parse(xhr.responseText).uuid, params.endpoint);
+                                        }
+                                    } else {
+                                        if (failure) {
+                                            try{
```
Comment (daniel-warren) [inline buildvu-client.js]: Two changes I would like to request for this section,
Could we rename "a" to something suitable to its purpose? For example parsedResponse.
Could the indentation at this section be corrected? (line 103, 105, 106, 107)

File: example.html [hunk 2]
```diff
@@ -55,10 +55,10 @@
 
     function successListener(e) {
         writeMessage(JSON.stringify(e));
-        writeMessage("\nConverted " + e.downloadUrl + "\n\n");
+        writeMessage("\nConverted " + e.previewUrl + "\n\n");
```
Comment (daniel-warren) [inline example.html]: Is there any reason why the example was changed back from downloadUrl?

Comment (niraliabhani) [reply example.html]: hey  @daniel-warren , I didnt realised I had made 3 commits instead of just one. I will check that and make proper commits

**Comment by niraliabhani** [comment]
@leonatherton @daniel-warren there was some confusion in my IDEA for commits and push in my local branch and remote branch .
I have tried to fix this .

**Pull request merged**